stdlib/include/builtins.isph

```diff
@@ -187,14 +187,14 @@ EXT varying int16 __pdivs_vi16(varying int16, varying int16);
 EXT varying int32 __pdivs_vi32(varying int32, varying int32);
 EXT varying int64 __pdivs_vi64(varying int64, varying int64);
 EXT varying int8 __pdivs_vi8(varying int8, varying int8);
-EXT uniform int16 __pdivus_ui16(uniform int16, uniform int16);
-EXT uniform int32 __pdivus_ui32(uniform int32, uniform int32);
-EXT uniform int64 __pdivus_ui64(uniform int64, uniform int64);
-EXT uniform int8 __pdivus_ui8(uniform int8, uniform int8);
-EXT varying int16 __pdivus_vi16(varying int16, varying int16);
-EXT varying int32 __pdivus_vi32(varying int32, varying int32);
-EXT varying int64 __pdivus_vi64(varying int64, varying int64);
-EXT varying int8 __pdivus_vi8(varying int8, varying int8);
+EXT uniform uint16 __pdivus_ui16(uniform uint16, uniform uint16);
```

Comment:
Hmm, looks like pdivus builtins are not used anywhere as well.

Reply:
Looks like a need a warning for unused declarations ;-)

Reply:
If __pdivus_* is really not used in stdlib.ispc, it can be removed from here.

Reply:
I have removed them.